sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -45,6 +35,24 @@
 from sklearn.utils import all_estimators
 from sklearn.exceptions import SkipTestWarning
 
+from sklearn.utils.estimator_checks import (
+    _NotAnArray,
+    _set_checking_parameters,
+    check_class_weight_balanced_linear_classifier,
+    check_classifier_data_not_an_array,
+    check_classifiers_multilabel_output_format_decision_function,
+    check_classifiers_multilabel_output_format_predict,
+    check_classifiers_multilabel_output_format_predict_proba,
+    check_estimator,
+    check_estimator_get_tags_default_keys,
+    check_estimators_unfitted,
+    check_fit_score_takes_y,
+    check_no_attributes_set_in_init,
+    check_regressor_data_not_an_array,
+    check_outlier_corruption,
+    set_random_state,
+)
```

Comment:
am I the only one who prefers a single line per import kinda style instead of this? :grin:

Reply:
The main pro of this convention is that it groups the symbols of a module or a submodule in one place, though there's no (automatic) hard checks on whether there are other imports from the same module or submodule somewhere else.

I prefer this style, but I agree that this adds yet another style in the codebase, which is something we might want to avoid. I don't have a strong opinion. :slightly_smiling_face:

Reply:
We could also just try to have the imports from the same module one after the other, which is what we try to do in other places anyway :D

Reply:
I am not preferring anything-anymore with `black` :)